website/content/docs/intro/usecases/what-is-a-service-mesh.mdx

```diff
@@ -0,0 +1,109 @@
+As service instances, either VMs or containers, come up and down, the mesh is aware of their state and availability. The ability to conduct _service discovery_ is the foundation to the other problems a _service mesh_ solves.
+
+As a service mesh is aware of the state of a service and its instances, the mesh can implement more intelligent and dynamic network routing.
+Many service meshes offer L7 traffic management capabilities. As a result, operators and developers can create powerful rules to direct network traffic as needed, such as load balancing, traffic splitting, dynamic failover, and custom resolvers.
+A service mesh's dynamic network behavior allows application owners to improve application resiliency and availability with no application changes.
+
+Implementing dynamic network behavior is critical as more and more applications are deployed across different cloud providers (multi-cloud) and private datacenters.
+Organizations may need to route network traffic to other infrastructure environments. Ensuring this traffic is secure is on top of mind for all organizations.
+Service meshes offer the ability to enforce network traffic encryption (mTLS) and authentication between all services. The _service mesh_ can automatically generate an SSL certificate for each service and its instances.
+The certificate authenticates with other services inside the mesh and encrypts the TCP/UDP/gRPC connection with SSL.
+
+Fine-grained policies that dictate what services are allowed to communicate with each other is another benefit of a _service mesh_.
+Traditionally, services are permitted to communicate with other services through firewall rules.
+The traditional firewall (IP-based) model is difficult to enforce with dynamic infrastructure resources with a short lifecycle and frequently recycling IP addresses.
+As a result, network administrators have to open up network ranges to permit network traffic between services without differentiating the services generating the network traffic. However, a _service mesh_ allows operators and developers to shift away from an IP-based model and focus more on service to service permissions.
+An operator defines a policy that only allows _service A_ to communicate with _service B_. Otherwise, the default action is to deny the traffic.
+This shift from an IP address-based security model to a service-focused model reduces the overhead of securing network traffic and allows an organization to take advantage of multi-cloud environments without sacrificing security due to complexity.
+
+## How Do You Implement a Service Mesh?
+
+Service meshes are commonly installed in Kubernetes clusters. There are also platform-agnostic service meshes available for non-Kubernetes-based workloads.
+For Kubernetes, most service mesh can be installed by operators through a [Helm chart](https://artifacthub.io/). Additionally, the service mesh may offer a CLI tool that supports the installation and maintenance of the service mesh.
```

Comment:
```suggestion
For Kubernetes, most service mesh can be installed by operators through a [Helm chart](https://helm.sh/). Additionally, the service mesh may offer a CLI tool that supports the installation and maintenance of the service mesh.
```
It might be better to link to the Helm website so that the reader can better understand what Helm is before exploring a chart registry like ArtifactHub.